### PR TITLE
Bump reported version for release 1.1.2

### PR DESCRIPTION
**Pull request by pgwillia**

### CODE REVIEW

File: app/models/jupiter_core.rb [hunk 1]
```diff
@@ -14,5 +14,5 @@ class LockedInstanceError < StandardError; end
   VISIBILITIES = [VISIBILITY_PUBLIC, VISIBILITY_PRIVATE, VISIBILITY_AUTHENTICATED].freeze
 
   # This'll do for the time being I suppose ¯\_(ツ)_/¯
```
Comment (murny) [inline app/models/jupiter_core.rb]: Probably be better to move the version stuff out of here.

Move this out into a `lib/jupiter/version.rb` file or something better

**Comment by murny** [comment]
Probably better to do these version bumbs when we actually cut the tag release. So you can reference the two in the git history

¯\_(ツ)_/¯

**Comment by pgwillia** [comment]
Call this a fire-drill. Think this process needs more discussion in general.

**Comment by murny** [comment]
100% agree :+1:

**Pull request closed without merging**